### PR TITLE
Document cross account resource management

### PR DESCRIPTION
**Pull request by A-Hilaly**
Issue #458 

Description of changes:
- Add cross account resource management documentation

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/contents/user-docs/authorization.md [hunk 1]
```diff
@@ -150,4 +150,74 @@ Some services may need an additional inline policy, specified in
 
 ## Cross-account resource management
 
-TODO
+ACK service controllers can manage resource in different AWS accounts. To enable and start using this feature, as an administrator, you will need to:
```
Comment (dbyron0) [inline docs/contents/user-docs/authorization.md:153]:
```suggestion
ACK service controllers can manage resources in different AWS accounts. To enable and start using this feature, as an administrator, you will need to:
```

Comment (A-Hilaly) [reply docs/contents/user-docs/authorization.md]: Fixed. Thanks @dbyron0!

File: docs/contents/user-docs/authorization.md [hunk 2]
```diff
@@ -150,4 +150,74 @@ Some services may need an additional inline policy, specified in
...
+
+### Setting up AWS accounts
+
+AWS Account administrators should create/configure IAM roles to allow ACK service controllers to assume Roles in different AWS accounts.
+
+For example, to allow account A (000000000000) to create s3 buckets in account B (111111111111) you can use the following commands
+```bash
+# Using account B credentials
+aws iam create-role --role-name s3FullAccess \
+  --assume-role-policy-document '{"Version": "2012-10-17","Statement": [{ "Effect": "Allow", "Principal": {"AWS": "arn:aws:iam::000000000000:root"}, "Action": "sts:AssumeRole"}]}'
```
Comment (jaypipes) [inline docs/contents/user-docs/authorization.md:168]: Let's not use `root` in any Principal please :)

File: docs/contents/user-docs/authorization.md [hunk 3]
```diff
@@ -150,4 +150,74 @@ Some services may need an additional inline policy, specified in
...
+aws iam attach-role-policy --role-name s3FullAccess \
+  --policy-arn 'arn:aws:iam::aws:policy/service-role/AmazonS3FullAccess'
+```
+
+### Map AWS Accounts with their associated Role ARNs
+
+After you will need to create a ConfigMap to associate each AWS Account ID with the role ARN that needs be assumed, in order to manage resource in that particular account.
```
Comment (jaypipes) [inline docs/contents/user-docs/authorization.md:175]: Please encapsulate `ConfigMap` with backticks here and throughout. Same with `Namespace` (which should be capitalized too)

Comment (jaypipes) [reply docs/contents/user-docs/authorization.md:175]: s/manage resource in/manage resources in/

File: docs/contents/user-docs/authorization.md [hunk 4]
```diff
@@ -150,4 +150,74 @@ Some services may need an additional inline policy, specified in
...
+metadata:
+  name: production
+  annotations:
+    services.k8s.aws/owner-account-id: 111111111111
+EOF
+```
+
+For existing namespaces you can also run:
+```bash
+kubectl annotate namespace services.k8s.aws/owner-account-id=111111111111
```
Comment (jaypipes) [inline docs/contents/user-docs/authorization.md:205]: You need to specify the name of the `Namespace` above... should be something like:

```
kubectl annotate namespaces production services.k8s.aws/owner-account-id=11111111
```

**Review by jaypipes — CHANGES_REQUESTED** [review]
Great stuff, @A-Hilaly thank you! A few requested changes inline please, though! :)

File: docs/contents/user-docs/authorization.md (hunk 4 above)
Comment (A-Hilaly) [reply docs/contents/user-docs/authorization.md]: Duhh, nice catch!

File: docs/contents/user-docs/authorization.md (hunk 3 above)
Comment (A-Hilaly) [reply docs/contents/user-docs/authorization.md]: Done

File: docs/contents/user-docs/authorization.md (hunk 2 above)
Comment (A-Hilaly) [reply docs/contents/user-docs/authorization.md]: Thanks for the pointer @jaypipes . Changed to use a role instead.

File: docs/contents/user-docs/authorization.md [hunk 5]
```diff
@@ -150,4 +150,74 @@ Some services may need an additional inline policy, specified in
...
+
+### Setting up AWS accounts
+
+AWS Account administrators should create/configure IAM roles to allow ACK service controllers to assume Roles in different AWS accounts.
+
+For example, to allow account A (000000000000) to create s3 buckets in account B (111111111111) you can use the following commands
+```bash
+# Using account B credentials
+aws iam create-role --role-name s3FullAccess \
+  --assume-role-policy-document '{"Version": "2012-10-17","Statement": [{ "Effect": "Allow", "Principal": {"AWS": "arn:aws:iam::000000000000:role/roleA-production"}, "Action": "sts:AssumeRole"}]}'
```
Comment (jaypipes) [inline docs/contents/user-docs/authorization.md:168]: :+1:

**Review by jaypipes — APPROVED** [review]
Nice :) thank you @A-Hilaly!

**Pull request merged**